charts/vela-core/templates/kubevela-controller.yaml

```diff
@@ -122,6 +122,7 @@ metadata:
   name: {{ include "kubevela.fullname" . }}
   namespace: {{ .Release.Namespace }}
   labels:
+    controller.oam.dev/name: vela-core
```

Comment:
according to the helm chart rule, you should add the label here: https://github.com/kubevela/kubevela/blob/master/charts/vela-core/templates/_helpers.tpl#L35-L44

BTW, don't forget add for vela-minimal

Reply:
> according to the helm chart rule, you should add the label here: https://github.com/kubevela/kubevela/blob/master/charts/vela-core/templates/_helpers.tpl#L35-L44
> 
> BTW, don't forget add for vela-minimal

As far as I Know, labels define in here will added to all types of resource (include configmap,secret) and so on. My point is to only add in controller's deployment.